documentation/decisions/003_component_api_style.md

```diff
@@ -0,0 +1,44 @@
+---
+status: Accepted
+date: 2023-02-03
+deciders: stephenjwatkins, OskiTheCoder, kevinalexliu
+---
+
+# Design component APIs based on common patterns that exist across popular libraries.
+
+## Context and Problem Statement
+
+Easy UI should have a set of basic standards and conventions for designing component APIs to deliver consistent DX and UX.
+
+## Decision Drivers
+
+- Scalability: As the project increaes in complexity, conventions should streamline the development process and provide reliability.
+- Intuitiveness: Easy UI component APIs should be approachable and easy to use.
+- Adaptability: The level of effort to adapt components to conform to API guidelines should be relatively low.
+
+## Considered Options
+
+- MUI
+
+- Shopify Polaris
+
+- Chakra UI
+
+- Ant Design
+
+- Twilio Paste
+
+- Adobe Spectrum
+
+- GitHub Primer
```

Comment:
do these need to have an extra space between them or can they be right next to each other?

```suggestion
- MUI
- Shopify Polaris
- Chakra UI
- Ant Design
- Twilio Paste
- Adobe Spectrum
- GitHub Primer
```